command/AudioFileHelper.js

```diff
@@ -21,15 +22,13 @@ class CommandHelper {
     return fileUrl;
   }
 
-  async getMobileFileLocation(articleId) {
+  //DONE
+  async getMobileFileMetadata(articleId) {
     // first check if we have this file in the DB
-    let fileUrl = await database.getMobileFileLocation(articleId);
-
-    if (!(await this.checkFileExistence(fileUrl))) {
-      fileUrl = '';
-    }
-
-    return fileUrl;
+    let result = {};
+    result = await database.getMobileMetadata(articleId);
+    //TODO: check if files exist.
```

Comment:
Todo in this patch or a followup?

Reply:
I think we should pause before implementing this based on what the bucket policy will be.  I'm thinking we might not need to check.